### ISSUE TITLE
Second Meeting

### ISSUE THREAD

**Issue by youngfoo19941** [youngfoo19941/New-Bus-Ticketing-System#2]
Okay guys, now we will start the second sprint planning meeting.

**Comment by freddywg1125** [comment]
oky. but what is the sprint about?

**Comment by YitTuckWeng** [comment]
yeah actually what is it about?

**Comment by youngfoo19941** [comment]
oh actually this sprint meeting is wanna to know your guys how was your work? did you guys meet any problem? its okay we can solve the problem together.

**Comment by NgZuhui** [comment]
my part done the ui and database already, I don't have face any problem

**Comment by YitTuckWeng** [comment]
@NgZuhui  that means you almost done the system ady?

**Comment by freddywg1125** [comment]
Wow so fast you done your ui and database, no error occur when you doing your database?

**Comment by NgZuhui** [comment]
ya, I done the function, but the ui design haven't done, got some thing to improve

**Comment by teychoonwei** [comment]
i also done the ui and the database but why i can't add data to the database?

**Comment by youngfoo19941** [comment]
i also got this problem but my problem is can't retrieve the data from the database

**Comment by freddywg1125** [comment]
Then young foo you should tell choon wei how to add the data into the database since you said you only got the problem of retrieve database.

**Comment by NgZuhui** [comment]
I think is sql command got problem, so young foo and choon wei please check you sql command

**Comment by teychoonwei** [comment]
ok

**Comment by youngfoo19941** [comment]
ok

**Comment by youngfoo19941** [comment]
so if don't have any question we will end the discussion, you all still have any question?

**Comment by teychoonwei** [comment]
no. i just have the database problem only

**Comment by NgZuhui** [comment]
no. i don't have any question

**Comment by freddywg1125** [comment]
I also do not have any question because I still doing it until never occur any error, but @teychoonwei zuhui already tell you the solution so did you solve it?

**Comment by YitTuckWeng** [comment]
@youngfoo19941 how was your retrieve problem? solved?

**Comment by teychoonwei** [comment]
later i will try

**Comment by youngfoo19941** [comment]
i also later try to solve

**Comment by youngfoo19941** [comment]
so now we will end the discusstion

**Comment by NgZuhui** [comment]
my part is no problem

**Comment by NgZuhui** [comment]
i totally agree. and we also can put some validation such as ic no number can not more  than or less than 12 digit
